app/views/comfy/admin/api_actions/_form.html.haml

```diff
@@ -115,6 +115,9 @@
         = label_tag 'Subject'
         =  text_field_tag "api_namespace[#{type}_attributes][#{index}][email_subject]", resource.email_subject, { class: "form-control form-control-sm" }
 
+      .form-group
```

Comment:
@Pralish is this change necessary? At the moment, I don't understand how its related to the feature